shuup/front/notify_events.py

```diff
@@ -81,12 +83,22 @@ class RefundCreated(Event):
 
 @receiver(order_creator_finished)
 def send_order_received_notification(order, **kwargs):
-    OrderReceived(
+    params = dict(
         order=order,
         customer_email=order.email,
         customer_phone=order.phone,
+        shop_email=None,
+        shop_phone=None,
         language=order.language
-    ).run()
+    )
+
+    if order.shop.contact_address:
+        params.update(dict(
```

Comment:
Can you add small test for this new shop phone/email option? Also test could cover the case when the contact address is not there.